app/manifest.template.json

```diff
@@ -21,6 +21,9 @@
     },
     "wrangle-current-tab": {
       "description": "__MSG_commandWrangleCurrentTab__"
+    },
+    "_execute_action": {
+      "description": "__MSG_commandWrangleOpenPopup__"
```

Comment:
There are Chrome- and [Firefox-specific](https://github.com/tabwrangler/tabwrangler/blob/3caa55d6e224142c0bcd3450aa5aea1aecd66154/webpack.config.js#L120-L143) configs in `webpack.config.js`. Can you move this into the Firefox-specific config so it is only added in FF? That way Chrome will not get duplicate shortcut keys.

Reply:
Thank you for pointing out! Done ✅